spk/emby/src/dsm-control.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/sh
+
+# Package
+PACKAGE="emby"
+DNAME="Emby"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+PATH="${INSTALL_DIR}/bin:${PATH}"
+USER="${PACKAGE}"
+USER_HOME="$(eval echo ~$USER)"
```

Comment:
i think this isn't even used at all, just a leftover from the nzbdrone installer...
i _think_ that can be deleted